Rozdział 10/kółko.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
```

Comment:
no i w tym pliku wiadomo wszystkie rzeczy z klikacza, ale to ich nie nanoś bo bez sensu w dwóch miejscach